File: app/src/main/java/com/example/esl/models/local/entities/Ulasan.kt
package com.example.esl.models.local.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "ulasan")
data class Ulasan(
    @PrimaryKey(autoGenerate = true) val id_ulasan: Int = 0,
    val id_users: Int,
    val id_penyewaan: Int,
    val ulasan: String,
    val rating: Int,
    val media_ulasan: String?,
    val tanggal_input: String
)

File: models/ulasan.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Ulasan extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      Ulasan.belongsTo(models.Users, { foreignKey: 'id_users' });
      Ulasan.belongsTo(models.Penyewaan, { foreignKey: 'id_penyewaan' });
    }
  }
  Ulasan.init({
    id_ulasan: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true,
    },
    id_users: DataTypes.INTEGER,
    id_penyewaan: DataTypes.INTEGER,
    ulasan: DataTypes.TEXT,
    rating: DataTypes.INTEGER,
    media_ulasan: DataTypes.STRING,
    tanggal_input: {
      type: DataTypes.DATE,
      defaultValue: DataTypes.NOW,
    }
  }, {
    sequelize,
    modelName: 'Ulasan',
  });
  return Ulasan;
};